src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -316,6 +319,14 @@ public void verifyMessageEditText(String title, String message, String userName)
     Assertion.assertEquals(editMessageWrapper.findElement(messageUserNameBy).getText(), userName);
   }
 
+  public void verifyMessageEditTextRenameDone(String title, String message, String userName) {
+    wait.forElementVisible(editMessageWrapper2);
+    WebElement msgBodyTextBox = editMessageWrapper2.findElement(messageBodyBy);
+    wait.forElementVisible(msgBodyTextBox);
+
+    Assertion.assertEquals(editMessageWrapper2.findElement(messageUserNameBy).getText(), userName);
```

Comment:
It's very useful to have Assertion in test level, not nested inside methods in PageObject. What do you think about refactoring this method just to return `editMessageWrapper2.findElement(messageUserNameBy).getText()` and make Assertion on test level?